packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -486,7 +487,7 @@ contract OPContractsManager is ISemver, Initializable {
                 _input.basefeeScalar,
                 _input.blobBasefeeScalar,
                 bytes32(uint256(uint160(_input.roles.batcher))), // batcherHash
-                30_000_000,
+                _input.gasLimit,
```

Comment:
We need to make this change in `OPContractsManagerInterop` also

Reply:
good call: https://github.com/ethereum-optimism/optimism/pull/12224/commits/eef95031abc7d667a53379e90cef6cec01e7b03e